test/e2e/go.mod

```diff
@@ -1,54 +1,56 @@
 module github.com/ovn-org/ovn-kubernetes/test/e2e
 
-go 1.20
+go 1.21
+
+toolchain go1.21.8
```

Comment:
why? I think your local go version is not aligning, its better to use one verison 1.21, 1.21.6 1.21.8 I see its all over the place... pick the version which is actually being used by k/k 1.29.2 (go check which version is used in that release and use that same golang version)

Reply:
yes my local go is 1.21.8, will correct all of these to be 1.21 which is what 1.29 k/k is on